ceph-build/build/setup_rpm

```diff
@@ -39,6 +39,19 @@ CentOS|Fedora|SUSE*|RedHatEnterpriseServer)
                 $SUDO yum install -y yum-utils
                 ;;
         esac
+        major_ver=$(lsb_release -rs | cut -f1 -d.)
+        case $(lsb_release -si) in
+            CentOS)
+                if [ $major_ver = 7 ]; then
+                    $SUDO yum install centos-release-scl
+                fi
+                ;;
+            RedHatEnterpriseServer)
```

Comment:
we don't build for RHEL at all, don't think this is necessary. CentOS7 should be fine, unless you feel strongly about leaving it here for granularity?

There is also no need to specify `$major_ver` either because we don't have anything that is not CentOS7

Maybe I would suggest just commenting that when needed, RHEL can be enabled with `rhel-server-rhscl-7-rpms`

Reply:
okay, i found that we are checking `RedHatEnterpriseServer` in the case . will drop these tests.